includes/Data/Patterns.php

```diff
@@ -50,16 +53,19 @@ protected static function get_theme_step_patterns() {
 						'replace' => true,
 					),
 					'homepage-1'  => array(
-						'active' => true,
-						'shown'  => true,
+						'active'        => true,
+						'shown'         => true,
+						'wonder_blocks' => 'home-1',
```

Comment:
This is the best approach in terms of accurate fallbacks, page publishing and child theme generation. I know it is a bit hacky, but we can always duplicate a Wonder homepage key but map it to a different pattern from WonderBlocks (this will create duplicates in the event that all API calls fail and we show the fallbacks), but if you look at it mathematically, no matter the approach you take, you will never have a 1 to 1 mapping without fallbacks being duplicated/missing, depending on the difference of patterns between WonderBlocks and an active theme.